src/unix/linux_like/mod.rs

```diff
@@ -616,6 +616,8 @@ pub const MS_LAZYTIME: ::c_ulong = 0x2000000;
 pub const MS_ACTIVE: ::c_ulong = 0x40000000;
 pub const MS_MGC_VAL: ::c_ulong = 0xc0ed0000;
 pub const MS_MGC_MSK: ::c_ulong = 0xffff0000;
+pub const OPEN_TREE_CLONE: ::c_uint = 0x01;
+pub const OPEN_TREE_CLOEXEC: ::c_uint = O_CLOEXEC as ::c_uint;
```

Comment:
Note for reviewers: these constants are typed as `uint` because they are used in the corresponding `open_tree()` syscall, which is [defined](https://github.com/torvalds/linux/blob/v6.1/fs/namespace.c#L2504) with an `unsigned flags` argument:
```
SYSCALL_DEFINE3(open_tree, int, dfd, const char __user *, filename, unsigned, flags)
```